sdk/ml/azure-ai-ml/azure/ai/ml/entities/_deployment/deployment_settings.py

```diff
@@ -139,7 +139,7 @@ def __init__(
         :type success_threshold: int
         :param timeout: timeout in seconds, defaults to 2
         :type timeout: int
-        :param period: [description], defaults to 10
+        :param period: How often (in seconds) to perform the probe, defaults to 10
         :type period: int
         :param initial_delay: How to to wait for the first probe, defaults to 10
```

Comment:
```suggestion
        :param initial_delay: How long (in seconds) to wait for the first probe, defaults to 10
```
While we're here - we should fix this typo too.